src/Compilers/CSharp/Test/Semantic/Semantics/IndexAndRangeTests.cs

```diff
@@ -193,14 +185,10 @@ struct S
         _ = this[r]; // 2
     }
 }";
+            // Note: due to our design for bound tree (LengthAccess is bound with placeholder) and CheckValue not being able
```

Comment:
>// Note: due to our design for bound tree (LengthAccess is bound with placeholder) and CheckValue not being able [](http://example.com/codeflow?start=12&length=112)

I think we should fix this instead of updating the baseline. See my other comment in TryBindIndexOrRangeImplicitIndexerParts